### ISSUE TITLE
XSS in search query

### ISSUE THREAD

**Issue by 3xocyte** [Creatiwity/wityCMS#146]
I noticed a reflected cross-site scripting vulnerability in the query parameter of the search function. You can trigger it by making a GET request like the following:

`http://localhost/witycms/search?query=<scRIpt>alert(0)</scRiPt>`

Cheers.

**Comment by JohanDufau** [comment]
Hello,

Thanks for the feedback!
We are going to investigate and publish a 0.6.2 fix soon.

See you,

**Comment by 3xocyte** [comment]
Hi Johan,

I haven't had a chance to recheck with an updated instance of wityCMS but I did check the filtering function you implemented. I found that it can still be bypassed. 

Eg, the payload: 
`<sc<script>ript>alert(0)</sc</script>ript> `
bypasses the check for "script".

Javascript can still be executed via allowed tags too, eg:
`<img src=a onerror=alert(0)>`

If you don't want to filter html tags, fixing XSS here is going to be really tricky. :/
